### PR TITLE
fix CVE-2023-25166 by resolving @sideway/formula to 3.0.1

### PR DESCRIPTION
**Pull request by jpcloureiro**
**Description**

Fix [CVE-2023-25166](https://github.com/advisories/GHSA-c2jc-4fpr-4vhg) by resolving `@sideway/formula` package to 3.0.1

### CODE REVIEW

**Review by tommasini — APPROVED** [review]
LGTM!

File: package.json [hunk 1]
```diff
@@ -90,6 +90,7 @@
     "**/json-schema": "^0.4.0",
     "@sentry/react-native/**/xmldom": "^0.6.0",
     "@sentry/**/simple-plist": "^1.3.1",
+    "@sideway/formula": "3.0.1",
```
[the remark this answers is not in the document]
Comment (jpcloureiro) [reply package.json]: I think we should. Good one 👍 
addressed in f1b7f11

**Review by gantunesr — APPROVED** [review]

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (gantunesr) [reply package.json]: I think we're trying to avoid it and keep versions fixed

[the remark this answers is not in the document]
Comment (jpcloureiro) [reply package.json:93]: Thanks for your input @gantunesr.
removed in af8ccb6

**Pull request merged**